config/main.py

```diff
@@ -4102,11 +4102,6 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
     click.secho("\nAfter running Logic to limit the impact", fg="cyan", underline=True)
     matched_items = [intf for intf in del_intf_dict if intf in add_intf_dict and del_intf_dict[intf] == add_intf_dict[intf]]
 
-    # Remove the interface which remains unchanged from both del_intf_dict and add_intf_dict
-    for item in matched_items:
```

Comment:
A few comments: 
1, Line 4099-4104 should be removed too.
2, Line 4111 should be removed too. 
3, Please rename the PR title to something like "[DPB] Remove the optimization logic for delete/add ports."